cloudwatch-exporter.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: cloudwatch-exporter
+  version: "0.15.3"
+  epoch: 0
+  description: Metrics exporter for Amazon AWS CloudWatch
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation:
+      license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-17-jre
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - maven
+      - openjdk-17
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/prometheus/cloudwatch_exporter
+      tag: v${{package.version}}
+      expected-commit: eb9f513167eb1bf2efeadbf67e9641b610d3dd4b
+
+  - runs: |
+      # CHECK-WHEN-VERSION-CHANGES: 0.15.3
+      # Address CVE-2022-1471
+      sed -i 's/1.33/2.0/g' pom.xml
```

Comment:
humm, this seems very fragile.  should we maybe have a tool which actually understands the XML and does the edits?

Reply:
Swapped it to just applying a patch.